### PR TITLE
 remove a todo which is out of date

### PR DESCRIPTION
**Pull request by hanxiaoshuai**
**What this PR does / why we need it**:
fix todo: move container.go to e2e/framework
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: test/e2e/framework/container.go [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2016 The Kubernetes Authors.
+Copyright 2018 The Kubernetes Authors.
```
Comment (deads2k) [inline test/e2e/framework/container.go]: don't change copyrights on existing files for moves

**Comment by deads2k** [comment]
I'm a little surprised to see this coming back.  What are you planning to do with it that requires it outside of the kubelet e2es?

File: test/e2e/framework/container.go [hunk 2]
```diff
@@ -24,18 +24,16 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/uuid"
 	podutil "k8s.io/kubernetes/pkg/api/v1/pod"
-	"k8s.io/kubernetes/test/e2e/framework"
 )
 
 // One pod one container
-// TODO: This should be migrated to the e2e framework.
```
Comment (hanxiaoshuai) [inline test/e2e/framework/container.go]: @deads2k I notice a todo here, do I misunderstand it?

Comment (deads2k) [reply test/e2e/framework/container.go]: > @deads2k I notice a todo here, do I misunderstand it?

I think the todo is out of date.  How about removing the todo, leaving the code where it is, and merging that instead?

Comment (hanxiaoshuai) [reply test/e2e/framework/container.go]: OK, I done it , Thanks.

**Pull request merged**